src/main/java/core/basesyntax/WorkWithBytes.java

```diff
@@ -1,7 +1,26 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithBytes {
     public void writeBytesToFile(String fileName, byte[] data) {
+        File file = createFile(fileName);
+        try {
+            Files.write(file.toPath(), data);
```

Comment:
```suggestion
           Files.write(Path.of(fileName), data);
```
try this way without creating file object

Reply:
done